utils/components.py

```diff
@@ -59,12 +59,23 @@ def list_models_in_directory_op(models_folder: str) -> List[str]:
     return models
 
 
-@dsl.component(
-    base_image=PYTHON_IMAGE,
-    install_kfp_package=False,
-    packages_to_install=["huggingface_hub"],
-)
-def huggingface_importer_op(repo_name: str, model_path: str = "/model"):
-    from huggingface_hub import snapshot_download
+@dsl.container_component
+def model_to_pvc_op(model: dsl.Input[dsl.Model], pvc_path: str = "/model"):
+    return dsl.ContainerSpec(
+        TOOLBOX_IMAGE,
+        ["/bin/sh", "-c"],
+        [f"cp -r {model.path}/* {pvc_path}"],
+    )
+
 
-    snapshot_download(repo_id=repo_name, cache_dir="/tmp", local_dir=model_path)
+@dsl.container_component
+def ilab_importer_op(repository: str, release: str, base_model: dsl.Output[dsl.Model]):
+    return dsl.ContainerSpec(
+        RHELAI_IMAGE,
+        ["/bin/sh", "-c"],
+        [
+            "echo 'Downloading model...' && "
+            + "echo '{}' > $HOME/config.yaml && "
+            + f"ilab  --config $HOME/config.yaml model download --repository {repository} --release {release} --model-dir {base_model.path}"
```

Comment:
```suggestion
            + f"ilab --config=DEFAULT model download --repository {repository} --release {release} --model-dir {base_model.path}"
```

Reply:
See below, we need the to force it to use our config. Other option is to generate it through `ilab config init` which is an interactive prompt...